### ISSUE TITLE
Activity Section Error and Edgecase

### ISSUE THREAD

**Issue by ajsarkar28** [department-of-veterans-affairs/va-mobile-app#7937]
For all visual elements in this ticket, reflect the designs in this [Figma file](https://www.figma.com/file/q4oTyzyLfYrZVz4CACzVLA/Personalization-Visual-Design---Working---VA-Mobile?type=design&node-id=3094-3373&mode=design&t=zB6JMXkiMjMDWDhc-4)

**Acceptance Criteria** 
- Activity section of the home screen reflects the logic and content in this [Mural](https://app.mural.co/t/adhoccorporateworkspace2583/m/adhoccorporateworkspace2583/1704381519703/cd6d78c5824a5fb0cc8b26597f0ad271072df8f2?sender=u4daa07e974be596b6b097258)

**Accessibility & Content** 
- All accessibility guidelines in previously linked Figma file have been followed
- All content is linked in this [Mural](https://app.mural.co/t/adhoccorporateworkspace2583/m/adhoccorporateworkspace2583/1704381519703/cd6d78c5824a5fb0cc8b26597f0ad271072df8f2?sender=u4daa07e974be596b6b097258)

**Comment by TKDickson** [comment]
Changing to just activity section (because an 'about you error and edge case' ticket exists - #7967). 

Tagging @ajsarkar28 if you want to re-combine them & delete the duplicate, as an FYI.

**Comment by TKDickson** [comment]
[Test cases link.](https://dsvavsp.testrail.io/index.php?/cases/view/63241) I _think_ the Cerner stuff for the home screen is figured out and it's just the health screen that's in flux, but should double-check before starting on this ticket/may need to update those cases.

**Comment by theodur** [comment]
Most of the work for this has been completed in the linked branch. The remainder of the work (handling error and edge cases for appointments and messages) can't be finished until the React Query migration for those features are merged into develop.

**Comment by TKDickson** [comment]
[Test run.](https://dsvavsp.testrail.io/index.php?/runs/view/5503&group_by=cases:section_id&group_order=asc)

One issue found - we're not checking for the appeals maintenance window. That's subissue 5 on #8578. Closing this ticket as testing complete.
